topics/transcriptomics/tutorials/rna-seq-viz-with-cummerbund/tutorial.md

```diff
@@ -116,53 +138,59 @@ For this tutorial, we are interested in CuffDiff's tested transcripts for differ
 
 # Filtering and sorting
 
-We now want to first highlight the most significant differentially expressed genes in our analysis, and then obtain informative visualizations.
+We now want to highlight those gene transcripts whose expression difference, the log<sub>2</sub>(fold_change), has been statistically assessed as both high and significant.
```

Comment:
We now want to highlight the gene transcripts whose expression difference, the log<sub>2</sub>(fold_change), has been statistically assessed to be both high and significant.

Reply:
or I think you can leave "statistically assessed as"